src/sentry/services/sentryshot/service.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+from sentry.models.sentryshot import SentryShot
+
+MAX_WAIT_TIME = 5
+logger = logging.getLogger("service:sentryshot")
+
+
+class SentryShotService:
+    def __init__(self, uuid):
+        self.sentryshot = SentryShot.objects.get(uuid=uuid)
+
+    def _start(self):
+        chrome_options = Options()
+        chrome_options.add_argument("--headless")
+        self.driver = webdriver.Chrome(options=chrome_options)
+
+    def generate_screenshot(self):
+        # TODO: @athena check gcs first to see if the image is already generated
+        try:
```

Comment:
do we ever want to retry this in case something is going wrong on the sentry side? like a regression where things are slow to load?

Reply:
i guess i'm wondering what happens if we are trying to send an alert and the image is too slow to load -- not send an image?

Reply:
There will be retry from the outside. I will be running this with a celery async task and plan to add retries to it.
In terms of what user sees, I'll handle failure in the `get` endpoint that they're hitting so if screenshot is available we give it to them if not we probably return a default image or something.